<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditGroupMappingDialog</class>
 <widget class="QDialog" name="EditGroupMappingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Group Mapping</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Available Columns</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTableWidget" name="tblMappings">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTableWidget" name="tblAlias">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btLoadMappings">
       <property name="toolTip">
        <string>Load mapping from a file</string>
       </property>
       <property name="statusTip">
        <string>Load mapping from a file</string>
       </property>
       <property name="text">
        <string>&amp;Load</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/document-open.svg</normaloff>:/icons/images/document-open.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btSaveMappings">
       <property name="toolTip">
        <string>Save mapping to a file</string>
       </property>
       <property name="statusTip">
        <string>Save mapping to a file</string>
       </property>
       <property name="text">
        <string>&amp;Save</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/document-save-as.svg</normaloff>:/icons/images/document-save-as.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mappings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btAddMapping">
       <property name="toolTip">
        <string>Add a new bin at the end of the list</string>
       </property>
       <property name="statusTip">
        <string>Add a new bin at the end of the list</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/list-add.svg</normaloff>:/icons/images/list-add.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btRemoveMappings">
       <property name="toolTip">
        <string>Remove selected bins from the list</string>
       </property>
       <property name="statusTip">
        <string>Remove selected bins from the list</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="icon">
        <iconset resource="ui.qrc">
         <normaloff>:/icons/images/list-remove.svg</normaloff>:/icons/images/list-remove.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditGroupMappingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditGroupMappingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
